spec/helpers/curation_concern_helper_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+describe CurationConcernHelper do
+  describe "#custom_h1" do
+    let(:gw) { GenericWork.new }
```

Comment:
nice work using `let` to assign local vars, and `.new` since this is just a GET to display a form